server/integration_tests/test_data/e2e_correlation_bugs/chart_config.json

```diff
@@ -14,16 +14,16 @@
                     },
                     "statVarKey": [
                       "Percent_Person_WithDiabetes_scatter",
-                      "Count_Person_BelowPovertyLevelInThePast12Months_HispanicOrLatino_scatter"
+                      "Count_Person_AbovePovertyLevelInThePast12Months_HispanicOrLatino_scatter"
```

Comment:
Make both "above" vs. "below" poverty use consistent strings.

"Number of hispanic or latino people above poverty line in the past year"